docs/metadata-format.md

```diff
@@ -1,379 +1,282 @@
+In your [data repository](glossary.md#data-repository) the metadata is maintained on an indicator-by-indicator basis. 
+
+By default the metadata fields are based on SDG SDMX detailed metadata concepts and the template metadata file in the data starter is based on these fields. It is recommended that you use the default fields as they follow the format that is requested by the UN and allow metadata to compared. However, the metadata can include any number of custom fields, as defined in a [schema file](https://github.com/open-sdg/open-sdg-data-starter/blob/develop/metadata_schema.yml) (see the "Schema" section below) in your data repository. However, 
+
+Whilst it is possible to use the SDG SDMX main metadata concepts (which are a level above the detailed concepts) e.g. **0. Indicator information**, the UNSD strongly recommends that the detailed concepts are used.
+
+Note: If you do choose to use the main metadata concepts, they should not be used in combination with the detailed metadata concepts.
+
+## Note about metadata file formats
+
+By default, Open SDG platforms expect the individual indicator metadata files to be uploaded in YML format to the "meta" folder. See this [example in the data starter repo](https://github.com/open-sdg/open-sdg-data-starter/blob/develop/meta/1-1-1.yml)
+
+### Indicator metadata forms
+
+To help with specifying indicator metadata, you can optionally enable user-friendly indicator metadata forms, which will allow you to fill in a form and then download a YML file which will be ready to upload to your platform.
+
+For more details on how to enable these forms, see the [indicator_metadata_form site configuration setting](configuration.md#indicator_metadata_form).
+
+## Note about empty metadata fields
+
+Due to the large number of metadata fields with some maybe not being applicable, empty metadata fields are hidden by default using `hide_empty_metadata: true` in the site configuration file.
+
+If you want metdata fields to show regardless, you should change this option to false. Alternatively, if you want some fields to hide by default but want to purposely not provide any information for other fields, you could populate the fields with some standard text e.g. "Not available"
+
+For more information about this configuration setting see the [hide_empty_metadata configuration option documentation](https://open-sdg.readthedocs.io/en/latest/configuration/#hide_empty_metadata).
 
 ## Note about translation keys
 
 Metadata values can either be filled in with normal text ("My field value") or with [translation keys](glossary.md#translation-keys) (my_translations.my_translation). In the examples below, we will try to demonstrate both possibilities.
 
 As an optional shorthand, if the translation key is in the `data` group, then the group can be omitted. For example, the translation key `data.female` can be written as simply `female`.
```

Comment:
Like with the indicator config, personally I would vote to remove this part about the data group.